app/Main.hs

```diff
@@ -1,5 +1,4 @@
-import TreasureKeeper (someFunc)
+import Treasure.Tui (executeTreasure)
 
 main :: IO ()
-main = someFunc
-
+main = executeTreasure
```

Comment:
Is it possible to create funnier name?.. Like, `monitorTreasure` (this is not that funny as well). Just an idea...

Reply:
`openTreasureChest`?